src/main/java/jpo/sdw/depositor/DepositorProperties.java

```diff
@@ -20,7 +20,7 @@ public class DepositorProperties implements EnvironmentAware {
 
    private static final String DEFAULT_GROUP_ID = "usdot.jpo.sdw";
    private static final String DEFAULT_KAFKA_PORT = "9092";
-   private static final String DEFAULT_DESTINATION_URL = "https://sdx-service.trihydro.com/api/deposit";
+   private static final String DEFAULT_DESTINATION_URL = "https://sdxdev-service.trihydro.com/api/deposit-multi";
```

Comment:
We'll want this default to point to the production service rather than dev

Reply:
We'll still want this pointing to the /deposit-multi endpoint to take advantage of the multiple object deposit. We'll just want it to target the production service: https://sdx-service.trihydro.com/api/deposit-multi

Reply:
Updated this to target the production deposit-multi endpoint